<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F1Fantasy</class>
 <widget class="QMainWindow" name="F1Fantasy">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>368</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>495</width>
    <height>368</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>1</width>
    <height>1</height>
   </size>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>1</horstretch>
     <verstretch>1</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>557</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="2">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item row="0" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QLabel" name="team_price">
          <property name="text">
           <string>Team Price</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="driver4">
          <property name="text">
           <string>Driver4</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="driver1">
          <property name="text">
           <string>Driver1</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="driver2">
          <property name="text">
           <string>Driver2</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QLabel" name="constructor">
          <property name="text">
           <string>Constructor</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="driver5">
          <property name="text">
           <string>Driver5</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QLabel" name="remaning_budget">
          <property name="text">
           <string>Remaning Price</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4" alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="driver3">
          <property name="text">
           <string>Driver3</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="next_race">
          <property name="text">
           <string>Next Race:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="recent_driver_changes">
          <property name="text">
           <string>Recent Driver Changes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="explain_htp">
          <property name="text">
           <string>Explanation on how to play</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="how_vac">
          <property name="text">
           <string>How values are calculated</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="edit_team">
          <property name="text">
           <string>Edit Team</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="refresh">
          <property name="text">
           <string>Refresh Data</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exit">
          <property name="text">
           <string>Exit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QScrollArea" name="chat_output">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::DefaultContextMenu</enum>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>190</width>
             <height>273</height>
            </rect>
           </property>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>220</width>
              <height>16</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Chat:</string>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="chat_input">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">placeholder=&quot;Type Message here&quot;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Type Message Here</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>615</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
